content/source/docs/enterprise/api/team-access.html.md

```diff
@@ -12,13 +12,15 @@ The team access APIs are used to associate a team to permissions on a workspace.
 
 ## List Team Access to Workspaces
 
-| Method | Path           |
-| :----- | :------------- |
-| GET | /team-workspaces |
+`GET /team-workspaces`
 
-### Parameters
+### Query Parameters
 
-- `filter[workspace][id]` (`string: <required>`) - The workspace ID for which to list the teams with access
+[These are standard URL query parameters](./index.html.html#query-parameters); remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
+
+Parameter               | Description
+------------------------|------------
+`filter[workspace][id]` | **Required** The workspace ID for which to list the teams with access.
```

Comment:
Should Required have a `.` here? The template has one.

Reply:
good catch.